en/22/lessoncomplete.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Wrapping It Up
+actions: ['checkAnswer', 'hints']
+material:
+  saveZombie: true
+  zombieBattle:
+    zombie:
+      lesson: 1
+    humanBattle: false
+    hideSliders: true
+    answer: 1
+---
+
+That's it, you've completed lesson 2!
+
+You can check out the demo to the right to see it in action. Go ahead, I know you can't wait until the bottom of this page 😉. Click a kitty to attack, and see the new kitty zombie you get!
+
+## JavaScript implementation
+
+Once we're ready to deploy this contract to Ethereum we'll just compile and deploy `ZombieFeeding` — since this contract is our final contract that inherits from `ZombieFactory`, and has access to all the public methods in both contracts.
+
+Let's look at an example of interacting with our deployed contract using JavaScript and web3.js:
```

Comment:
Just wanted to double-check with you: are you folks using web3.js and does the code below work for you?